src/components/pokeballSelector.html

```diff
@@ -2,6 +2,13 @@
     <div class="card-header p-0" data-toggle="collapse" href="#pokeballSelectorBody">
         <span>Poké Balls</span>
     </div>
+    <!-- ko if: QuestLineHelper.isQuestLineCompleted('Tutorial Quests') -->
```

Comment:
We might want to unlock this a bit later. I am not sure people want to use it this early, and they already unlock a lot of stuff this early, that they need to understand.
But i am not sure

Reply:
I didn't want to restrict it for experienced players who may want it available early, I only put it behind the tutorial to prevent people messing up the catching walkthrough. We can revisit the unlock condition later, if it seems like this is confusing people?